src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -196,6 +196,10 @@ public static Object read(final RedisInputStream is) {
     return process(is);
   }
 
+  public static String readLine(RedisInputStream inputStream) {
```

Comment:
What's the purpose of this new method? Can't we just do inputStream.readLine() in https://github.com/xetorthio/jedis/pull/985/files#diff-99ff5b9f483253e37476a66788c13ec6R104